.gitignore

```diff
@@ -33,13 +33,15 @@ yarn-debug.log*
 *.log
 .envrc
 terraform/app_secrets.yml
+*token*
```

Comment:
This is too heavy handed. We have `app/services/dttp/access_token_service.rb` which will be ignored. We can't ignore everything with `token` in it.

Reply:
cool... noted.

Reply:
now relexed..... in hindsight, it was heavy handed. i guess i was too cautious